src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas3/DomainEventService.kt

```diff
@@ -203,7 +204,7 @@ class DomainEventService(
         occurredAt = domainEvent.occurredAt.atOffset(ZoneOffset.UTC),
         createdAt = OffsetDateTime.now(),
         data = objectMapper.writeValueAsString(domainEvent.data),
-        service = "CAS3",
+        service = ServiceName.temporaryAccommodation.name,
```

Comment:
I'm pretty sure "CAS3" is correct in this case, looking at the discriminator in the yml